sdk/src/client/secret/ledger_nano.rs

```diff
@@ -326,21 +325,19 @@ impl SecretManage for LedgerSecretManager {
                         // The outputs in the essence already are sorted
                         // at this place, so we can rely on their order and don't have to sort it again.
                         'essence_outputs: for output in essence.outputs().iter() {
```

Comment:
In the ISA we put the remainder at last position, so rev() should be faster
```suggestion
                        'essence_outputs: for output in essence.outputs().iter().rev() {
```

Reply:
Ok but need to be careful with the index then

Reply:
Ah yeah we can also just leave it as is, max is like ~20 rounds anyways